### PR TITLE
#165305715 Install and setup sass pre-processing on the application

### PR DESCRIPTION
**Pull request by IssaIan**
### What does this PR do?
Install and setup sass pre-processing on the application to enables the use of both .scss and .css files on the client
### Description of task to be completed
- Install sass dependencies
- Update webpack configurations to enable use of .css and .scss files
- Add sample css styling to test
- Install bootstrap and react-bootstrap dependencies
### Screenshot
image
### Relevant Pivotal Tracker Story
[#165305715](https://www.pivotaltracker.com/n/projects/2326460/stories/165305715)

### CODE REVIEW

File: src/css/App.css [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.test {
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    background: rgba(3, 578, 76, 4);
+  }
```
Comment (Ogutu-Brian) [inline src/css/App.css]: Hello @IssaIan ,
Would you add a new line at the end of file

**Review by Ogutu-Brian — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 2]
```diff
@@ -61,3 +61,4 @@ typings/
 .next
 .DS_Store
 dist/
+package-lock.json
```
Comment (Ogutu-Brian) [inline .gitignore]: Please add a new line at the end of file.

**Review by Ogutu-Brian — CHANGES_REQUESTED** [review]

File: src/css/App.css (hunk 1 above)
Comment (IssaIan) [reply src/css/App.css]: made the changes requested

File: .gitignore (hunk 2 above)
Comment (IssaIan) [reply .gitignore]: CSS files do not require new lines at the end of the file. Error is thrown by github when one is applied. Check how eslint handles this @Ogutu-Brian.

**Review by Ogutu-Brian — APPROVED** [review]
LGTM

**Review by Amoswachira — APPROVED** [review]

**Review by matthenge — APPROVED** [review]

**Review by a-braham — APPROVED** [review]
LGTM

File: src/css/App.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.test {
```
Comment (koechkevin) [inline src/css/App.css]: rename the folder  to styles

**Review by koechkevin — CHANGES_REQUESTED** [review]
@IssaIan importing `.scss` files in any file apart from index.js makes the tests to fail. Please fix
Screenshot 2019-05-28 at 20 39 47

File: src/css/App.css (hunk 3 above)
Comment (IssaIan) [reply src/css/App.css]: Made the requested changes.

**Review by koechkevin — APPROVED** [review]

**Pull request merged**